### PR TITLE
Nov 27th stream feedback

### PR DESCRIPTION
**Pull request by Vivelin**
- Added bosses as separate entities, and reworked the boss/dungeon clearing voice commands;
- Added responses for asking hints about a location you already cleared;
- Changed clearing a dungeon to no longer complain about treasure count when you've been manually clearing treasure (without clearing the associated locations)
- Various config changes, including:
  - Removing "Love" because it seems to cause a lot of mistracks

### CODE REVIEW

File: src/Randomizer.SMZ3.Tracking/tracker.json [hunk 1]
```diff
@@ -471,7 +407,8 @@
       "Column": 6,
       "Row": 4,
       "WhenTracked": {
-        "1": [ "You've got mail!" ]
+        "1": [ "You've got mail!" ],
+        "2": null
```
Comment (MattEqualsCoder) [inline src/Randomizer.SMZ3.Tracking/tracker.json:411]: Was this change intentional? Just making sure this wouldn't cause a null error.

Comment (MattEqualsCoder) [reply src/Randomizer.SMZ3.Tracking/tracker.json:411]: nvm, looks like it should be fine

**Review by MattEqualsCoder — APPROVED** [review]

File: src/Randomizer.SMZ3.Tracking/tracker.json (hunk 1 above)
Comment (Vivelin) [reply src/Randomizer.SMZ3.Tracking/tracker.json:411]: Yup! I specifically detect `null` and take it to mean "use the default responses"
